### PR TITLE
Added Dashboard to the user Docs

### PR DESCRIPTION
**Pull request by Bangalisch**
Added 1.5 Dashboard page to documents. Included img in correct folder and added the new page to the menu structure.

### CODE REVIEW

**Comment by Bangalisch** [comment]
Took the vid done by dennis, converted it to a GIF for ez display. 
If there's a better video, please upload and attach. 

Otherwise close #1082

**Comment by pavlenex** [comment]
@Bangalisch there's a broken link so the build isnt' passing the linkcheck, can you address that?

```
Getting links from: http://localhost:8080/Dashboard/
[27](https://github.com/btcpayserver/btcpayserver-doc/runs/5958330042?check_suite_focus=true#step:10:27)
├─BROKEN─ http://localhost:8080/Apps.md/#crowdfunding-app (HTTP_404)
```

**Comment by Bangalisch** [comment]
Added the Tool tip as per dennis his comment. 

On the POS data, I have not included that ? I've included the Crowdfund as that is available ? The POS app, doesn't show up on the dash :)

File: docs/Dashboard.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Dashboard 
+
+:::tip
+Balance graph and recent transaction are only available for on-chain wallets, where we don't have any data that's special for Lightning.
```
Comment (pavlenex) [inline docs/Dashboard.md:4]:
```suggestion
Balance graph and recent transaction are currently only available for an on-chain payment method. Information about Lightning network balances and transactions is on to-do.
```

File: docs/Dashboard.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+:::
+
+This page will tell you about the Dashboard panel in your **BTCPay user interface**.
+As of BTCPay server v1.5, we've added new panels to the Dashboard page.
```
Comment (pavlenex) [inline docs/Dashboard.md:8]:
```suggestion
BTCPay Server version 1.5.0 introduced a new dashboard concept that features several tiles that will help with the initial setup, better understand the store's data and manage refunds and payouts with ease.
```

File: docs/Dashboard.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Dashboard 
+
+:::tip
+Balance graph and recent transaction are only available for on-chain wallets, where we don't have any data that's special for Lightning.
+:::
+
+This page will tell you about the Dashboard panel in your **BTCPay user interface**.
+As of BTCPay server v1.5, we've added new panels to the Dashboard page. 
+Not only will it now take you through the initial setup of your store's wallet, but once you are set up, you get basic analytics of your store!
+
```
Comment (pavlenex) [inline docs/Dashboard.md:10]: Already mentioned this in the paragraph above 😄 
```suggestion
```

File: docs/Dashboard.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+There are five main sections to the Dashboard. 
+
+* Wallet Balance 
+    This is the current store [Wallet](Wallet.md) balance. 
+    The view shows a balance graph by Week, Month, or year. 
+
+![BTCPay Server Navigation](./img/dashboard/wallet-view.jpg "BTCPay Server Navigation")
+
+* TX Activity
```
Comment (pavlenex) [inline docs/Dashboard.md:23]:
```suggestion
* Transaction activity 
```

File: docs/Dashboard.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+There are five main sections to the Dashboard. 
+
+* Wallet Balance 
+    This is the current store [Wallet](Wallet.md) balance. 
+    The view shows a balance graph by Week, Month, or year. 
+
+![BTCPay Server Navigation](./img/dashboard/wallet-view.jpg "BTCPay Server Navigation")
+
+* TX Activity 
+    Quickly manage Pending Payouts, view recent TXs, and overview outstanding [refunds](Refund.md)
```
Comment (pavlenex) [inline docs/Dashboard.md:24]:
```suggestion
    Quickly manage Pending Payouts, view recent transactions, and overview outstanding [refunds](Refund.md)
```

File: docs/Dashboard.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+![BTCPay Server Navigation](./img/dashboard/tx-activity-view.jpg "BTCPay Server Navigation")
+
+* Recent Transactions
+    The last five transactions are shown.
+
+![BTCPay Server Navigation](./img/dashboard/recent-tx-view.jpg "BTCPay Server Navigation")
+
+* Recent Invoices
+    The last five invoices are shown with their corresponding status and value.
+    It also allows you to go directly to all [invoices](/Invoices.md).
```
Comment (pavlenex) [inline docs/Dashboard.md:35]:
```suggestion
    It also allows you to directly access and manage a particular [invoice](/Invoices.md). 
```

File: docs/Dashboard.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+![BTCPay Server Navigation](./img/dashboard/recent-tx-view.jpg "BTCPay Server Navigation")
+
+* Recent Invoices
+    The last five invoices are shown with their corresponding status and value.
+    It also allows you to go directly to all [invoices](/Invoices.md). 
+
+![BTCPay Server Navigation](./img/dashboard/recent-invoice-view.jpg "BTCPay Server Navigation")
+
+* Current active [Crowdfund](Apps.md#Crowdfunding-App)
+    This tile shows the current active Crowdfunds, including their top-ranked items.
```
Comment (pavlenex) [inline docs/Dashboard.md:40]:
```suggestion
    This tile shows the current active crowdfunds, including their top-ranked items. 
```

File: docs/Dashboard.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+* Recent Invoices
+    The last five invoices are shown with their corresponding status and value.
+    It also allows you to go directly to all [invoices](/Invoices.md). 
+
+![BTCPay Server Navigation](./img/dashboard/recent-invoice-view.jpg "BTCPay Server Navigation")
+
+* Current active [Crowdfund](Apps.md#Crowdfunding-App)
+    This tile shows the current active Crowdfunds, including their top-ranked items. 
+    When more than one active Crowdfund is happening, the panels will show below the initial one.
```
Comment (pavlenex) [inline docs/Dashboard.md:41]: using tiles over panels for consistency.
```suggestion
    When more than one active crowdfund is happening, the tiles will show below the initial one. 
```

File: docs/Dashboard.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+* Recent Invoices
+    The last five invoices are shown with their corresponding status and value.
+    It also allows you to go directly to all [invoices](/Invoices.md). 
+
+![BTCPay Server Navigation](./img/dashboard/recent-invoice-view.jpg "BTCPay Server Navigation")
+
+* Current active [Crowdfund](Apps.md#Crowdfunding-App)
+    This tile shows the current active Crowdfunds, including their top-ranked items. 
+    When more than one active Crowdfund is happening, the panels will show below the initial one. 
+    There's easy access to Crowdfund management or to see all the perks you've set from these panels.
```
Comment (pavlenex) [inline docs/Dashboard.md:42]:
```suggestion
    There's easy access to crowdfund management or to see all the perks you've set from these panels. 
```

**Review by pavlenex — COMMENTED** [review]
Great work, added several comments :)

File: docs/Dashboard.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
+# Dashboard 
+
+:::tip
+Balance graph and recent transaction are currently only available for an on-chain payment method. Information about Lightning network balances and transactions is on to-do.
+:::
+
+BTCPay Server version 1.5.0 introduced a new dashboard concept that features several tiles that will help with the initial setup, better understand the store's data and manage refunds and payouts with ease.
+![BTCPay Server Navigation](./img/dashboard/dashboardgif.gif "BTCPay Server Navigation")
```
Comment (pavlenex) [inline docs/Dashboard.md:8]:
```suggestion

![BTCPay Server Navigation](./img/dashboard/dashboardgif.gif "BTCPay Server Navigation")
```

File: docs/Dashboard.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+## What do the tiles tell?
```
Comment (pavlenex) [inline docs/Dashboard.md:10]:
```suggestion
##  Dashboard tiles
```

File: docs/Dashboard.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+
+There are five main sections to the Dashboard. 
+
+* Wallet Balance
```
Comment (pavlenex) [inline docs/Dashboard.md:14]:
```suggestion
### Wallet Balance 
```

File: docs/Dashboard.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+BTCPay Server version 1.5.0 introduced a new dashboard concept that features several tiles that will help with the initial setup, better understand the store's data and manage refunds and payouts with ease.
+![BTCPay Server Navigation](./img/dashboard/dashboardgif.gif "BTCPay Server Navigation")
+
+## What do the tiles tell?
+
+There are five main sections to the Dashboard. 
+
+* Wallet Balance 
+    This is the current store [Wallet](Wallet.md) balance. 
+    The view shows a balance graph by Week, Month, or year.
```
Comment (pavlenex) [inline docs/Dashboard.md:16]:
```suggestion
This is the current store [Wallet](Wallet.md) balance.  The view shows a balance graph by week, month, or a year. 
```

File: docs/Dashboard.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+
+There are five main sections to the Dashboard. 
+
+* Wallet Balance 
+    This is the current store [Wallet](Wallet.md) balance. 
+    The view shows a balance graph by Week, Month, or year. 
+
+![BTCPay Server Navigation](./img/dashboard/wallet-view.jpg "BTCPay Server Navigation")
+
+* Transaction activity
```
Comment (pavlenex) [inline docs/Dashboard.md:20]:
```suggestion
### Transaction activity 
```

File: docs/Dashboard.md [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+There are five main sections to the Dashboard. 
+
+* Wallet Balance 
+    This is the current store [Wallet](Wallet.md) balance. 
+    The view shows a balance graph by Week, Month, or year. 
+
+![BTCPay Server Navigation](./img/dashboard/wallet-view.jpg "BTCPay Server Navigation")
+
+* Transaction activity 
+    Quickly manage Pending Payouts, view recent transactions, and overview outstanding [refunds](Refund.md)
```
Comment (pavlenex) [inline docs/Dashboard.md:21]:
```suggestion
Quickly manage pending payouts, view recent transactions, and overview outstanding [refunds](Refund.md)
```

File: docs/Dashboard.md [hunk 16]
```diff
@@ -0,0 +1,46 @@
...
+
+![BTCPay Server Navigation](./img/dashboard/tx-activity-view.jpg "BTCPay Server Navigation")
+
+* Recent Transactions
```
Comment (pavlenex) [inline docs/Dashboard.md:25]:
```suggestion
### Recent Transactions
```

File: docs/Dashboard.md [hunk 17]
```diff
@@ -0,0 +1,46 @@
...
+    The last five transactions are shown.
```
Comment (pavlenex) [inline docs/Dashboard.md:26]:
```suggestion
Showcasing the five most recent transactions that arrived to your on-chain wallet. 
```

File: docs/Dashboard.md [hunk 18]
```diff
@@ -0,0 +1,46 @@
...
+
+![BTCPay Server Navigation](./img/dashboard/recent-tx-view.jpg "BTCPay Server Navigation")
+
+* Recent Invoices
```
Comment (pavlenex) [inline docs/Dashboard.md:30]:
```suggestion
### Recent invoices
```

File: docs/Dashboard.md [hunk 19]
```diff
@@ -0,0 +1,46 @@
...
+    The last five invoices are shown with their corresponding status and value.
+    It also allows you to directly access and manage a particular [invoice](/Invoices.md).
```
Comment (pavlenex) [inline docs/Dashboard.md:32]:
```suggestion
The five  most recent invoices are shown with their corresponding status and value allowing you to quickly access and manage a particular [invoice](/Invoices.md). 
```

File: docs/Dashboard.md [hunk 20]
```diff
@@ -0,0 +1,46 @@
...
+
+![BTCPay Server Navigation](./img/dashboard/recent-tx-view.jpg "BTCPay Server Navigation")
+
+* Recent Invoices
+    The last five invoices are shown with their corresponding status and value.
+    It also allows you to directly access and manage a particular [invoice](/Invoices.md). 
+
+![BTCPay Server Navigation](./img/dashboard/recent-invoice-view.jpg "BTCPay Server Navigation")
+
+* Current active [Crowdfund](Apps.md#Crowdfunding-App)
```
Comment (pavlenex) [inline docs/Dashboard.md:36]:
```suggestion
### Current active crowdfund
```

File: docs/Dashboard.md [hunk 21]
```diff
@@ -0,0 +1,46 @@
...
+    This tile shows the current active crowdfunds, including their top-ranked items. 
+    When more than one active crowdfund is happening, the tiles will show below the initial one. 
+    There's easy access to crowdfund management or to see all the perks you've set from these panels.
```
Comment (pavlenex) [inline docs/Dashboard.md:39]:
```suggestion
This tile shows the current active crowdfunds, including their top-ranked items/perks. When more than one crowdfunding app is active crowdfund is, the tiles will show below the initial one. That's an easy way to manage your active crowdfund campaigns and see all the perks and how they're performing. 
```

File: docs/Walkthrough.md [hunk 22]
```diff
@@ -54,15 +54,24 @@ Learn more about all BTCPay notifications [here](./Notifications.md).
  
 ## Dashboard
 
-Right now, the dashboard only shows the primary tasks of setting up BTCPay Server, more to come in future releases.
+In dashboard you will see the stores wallet balance, a overview of invoices and quickly see your crowdfunds top perks. 
+There's 5 main tiles in the Dashboard.
+
+* A quick view on Wallet Balance
+* TX activity and payouts
```
Comment (pavlenex) [inline docs/Walkthrough.md:61]:
```suggestion
* Transaction and payout activity
```

File: docs/Walkthrough.md [hunk 23]
```diff
@@ -54,15 +54,24 @@ Learn more about all BTCPay notifications [here](./Notifications.md).
  
 ## Dashboard
 
-Right now, the dashboard only shows the primary tasks of setting up BTCPay Server, more to come in future releases.
+In dashboard you will see the stores wallet balance, a overview of invoices and quickly see your crowdfunds top perks. 
+There's 5 main tiles in the Dashboard.
+
+* A quick view on Wallet Balance
```
Comment (pavlenex) [inline docs/Walkthrough.md:60]:
```suggestion
* A quick view on the wallet balance
```

**Pull request merged**